doc/source/rllib-algorithms.rst

```diff
@@ -5,40 +5,42 @@ RLlib Algorithms
 
     Check out the `environments <rllib-env.html>`__ page to learn more about different environment types.
 
-Feature Compatibility Matrix
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Available Algorithms - Overview
```

Comment:
Changed the title to something simpler. Added transformer support links to respective algos.

Reply:
Can we split doc changes into a separate PR?

Reply:
done, separate PR now